files/galaxy/tpv/users.yml

```diff
@@ -6,7 +6,8 @@ users:
   andreas.kalagasidis@gmail.com:
     rules:
       - id: user-specific-notebook
-        if: 'interactive_tool_jupyter_notebook' in tool.id
+        if: |
+          'interactive_tool_jupyter_notebook' in tool.id
         cores: 1
```

Comment:
Wow, this is wired, shouldn't it be the same from YAML perspective?
Thanks for fixing!

Reply:
My best guess is that it was due to the single-quoted string.